organization/_posts/2017-12-13-packaging.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "HSF Packaging Group Meeting #16, December 13, 2017"
+layout: default
+---
+
+# HSF Packaging Group Meeting #16, December 13, 2017
+
+#### *Present*: Graeme Stewart, Ben Morgan, Jakob Blomer, Guilherme Amadio, Patrick Gartung, Javier Cervantes Villanueva, Rafal Pacholek, Chris Green, Pere Mato, Geri Ganis, Rafal, Xavi?, Oana Boeriu
```

Comment:
"Xavi?" was on vidyo, so I really don't know who they are. Maybe just change this to "Xavi (vidyo)" and we mention in the minutes announcement that we didn't know everyone who was on vidyo.